src/components/SkilCard/SkilCard.tsx

```diff
@@ -0,0 +1,69 @@
+/** @jsx jsx */
+import { jsx, css } from '@emotion/core'
+import { FC } from 'react'
+
+export type Props = {
+  icon: any
+  curriculum: string
+  explanation: string
+  maincolor: string
+  subcolor: string
+}
```

Comment:
Reactコンポーネントを受け取りたいときはanyではなく`ReactElement`を使用します。

```suggestion
/** @jsx jsx */
import { jsx, css } from '@emotion/core'
import { FC, ReactElement } from 'react'

export type Props = {
  icon: ReactElement
  curriculum: string
  explanation: string
  maincolor: string
  subcolor: string
}
```